### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by JkChenAcmer** [penny4860/tf-eager-yolo3#18]
Although the loss of the training set goes down, the loss of the valid set goes up.What does yolo net do to prevent overfitting.

**Comment by penny4860** [comment]
Use more training data, or use a regularization method.
